go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/googleapis/gax-go/v2 v2.0.5
 	go.opencensus.io v0.23.0
 	golang.org/x/oauth2 v0.0.0-20210805134026-6f1e6394065a
+	golang.org/x/tools v0.1.5 // indirect
```

Comment:
Please revert this change. It was because of the `go get` I asked you to run.

Reply:
reverted